test/test.js

```diff
@@ -1,47 +1,139 @@
-const mongoose = require('mongoose');
-const db = require('../db/index');
-
-let chai = require('chai');
-let chaiHttp = require('chai-http');
-let app = require('../server/index');
+let chai = require("chai");
+let chaiHttp = require("chai-http");
+let app = require("../server/index");
 
 let should = chai.should();
 chai.use(chaiHttp);
 
-describe('/GET comments', () => {
-  it('it should GET all the comments', (done) => {
-    chai.request(app)
-        .get('/comments')
-        .end((err, res) => {
-              res.should.have.status(200);
-              res.body.data.should.be.a('array');
-              res.body.data.length.should.be.gt(36);
-              res.body.data.length.should.be.below(1000);
-          done();
-        });
+// I should probably be using a seperate test database for these tests, but I don't feel like coding it right now, maybe later
+
+// ObjectID of comment that exists
+let existingComment = {
+  "user_id": 895597,
+  "song_id": 8888452,
+  "content": "Dolor esse velit cupidatat enim non tempor consequat.",
+  "time_stamp": 93,
+  "comment_id": "5fc367e0d17509f5cb743606"
+};
+
```

Comment:
So I get these are the mockdata, but shouldn't there be a point where you say before each (set the test environment) I don't know how this mockdata gets retrieved.

Reply:
I'm testing the same database as the app, and I already loaded in data, which I just looked up. So you are correct, the test file won't work if the database is empty (not a very good thing--I should be adding it in in the test file).